packages/features/README.md

```diff
@@ -27,10 +27,10 @@ This library uses the global [fetch](https://fetch.spec.whatwg.org/) function fo
 ``` javascript
 require('isomorphic-fetch');
 
-const { Service } = require('@ogcapi-js/features');
+const { FeatureService } = require('@ogcapi-js/features');
```

Comment:
I changed the name because I think `FeatureService` is more specific than `Service`. In addition it will avoid import aliases for consumers, which want to use the features and the process package together.

Reply:
I have checked the latest release (`0.5.0`) and this breaking change has already been implemented there, but the corresponding README was not updated accordingly. This change does this.

Reply:
Hmm, I am not aware of that change. I will still plan to release it as v1.